### ISSUE TITLE
[Bug]: Query growing data gets empty result on CI

### ISSUE THREAD

**Issue by ThreadDao** [milvus-io/milvus#12779]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

https://ci.milvus.io:18080/jenkins/blue/organizations/jenkins/milvus-ha-ci/detail/PR-12772/2/pipeline
```
[2021-12-06T07:31:55.085Z] [2021-12-06 07:31:42 - DEBUG - ci_test]: (api_request)  : [Collection.insert] args: [    int64  float                                       float_vector

[2021-12-06T07:31:55.085Z] 0       0    0.0  [0.10949418413493775, 0.00942660587660462, 0.1...

[2021-12-06T07:31:55.085Z] 1       1    1.0  [0.13349830221636338, 0.10102771046144066, 0.0...

[2021-12-06T07:31:55.085Z] 2       2    2.0  [0.11931290577075418, 0.026581915422442235, 0....

[2021-12-06T07:31:55.085Z] 3       3    3.0  [0.102618......, kwargs: {'timeout': 20} (api_request.py:55)

[2021-12-06T07:31:55.085Z] [2021-12-06 07:31:42 - DEBUG - ci_test]: (api_response) : (insert count: 100, delete count: 0, upsert count: 0, timestamp: 429595270198853633)  (api_request.py:27)

[2021-12-06T07:31:55.085Z] [2021-12-06 07:31:43 - DEBUG - ci_test]: (api_request)  : [Collection.query] args: ['int64 in [1]', None, None, 20], kwargs: {} (api_request.py:55)

[2021-12-06T07:31:55.085Z] [2021-12-06 07:31:43 - DEBUG - ci_test]: (api_response) : []  (api_request.py:27)
```

**Comment by xige-16** [comment]
I have reproduced this problem. If two growing semgnets are inserted, and the query result is empty, the growing segmentID of the query is the same, that is to say, a segment is missed in the query.

**Comment by xige-16** [comment]
When multiple flowgraphs update query node meta at the same time, some segment information may be overwritten

**Comment by xiaofan-luan** [comment]
we are planning to solve or those related issue by not supporting complicated load collection, release partition, load partition  behaviors, will be in soon

**Comment by xige-16** [comment]
the issue has been fixed by pr#12912
